doc/changelog/06-Ltac2-language/16740-tac2-preterm.rst

```diff
@@ -0,0 +1,4 @@
+- **Added:**
+  quotation and syntax class for preterms
```

Comment:
```suggestion
  quotation and syntax class for :ref:`preterms <preterm>`
```

and add the link into `ltac2.rst` just before this paragraph: "In the special case where Ltac2 antiquotations appear inside a Coq term"

Reply:
While that was the only reference prior to this PR, I think the right place to link is more likely 
~~~
- ``preterm``, which parses Coq terms and produces a value which must
  be typechecked with ``Constr.pretype`` (type ``Init.preterm``).
~~~
in the "built-in quotations" list or
~~~
  :n:`preterm {? ( {+, @scope_key } ) }`
    Parses a non-typechecked :token:`term`. Like :n:`constr` above, this class
    accepts a list of notation scopes with the same effects.
~~~
in the "syntactic classes" section.

Reply:
Or add some built-in type section

Reply:
> I think the right place to link is more likely ... in the "syntactic classes" section.

Sure, makes sense.

> Or add some built-in type section

Eventually.

Reply:
done